spm_results_export.m

```diff
@@ -28,14 +27,12 @@ function spm_results_export(SPM,xSPM,TabDat)
     if exist(fullfile(SPM.filepath,'html')) ~= 7 

```

Comment:
To make the code safer, could you move l. 27-32 (deleting previously computed `html` folder) to l.26 of `nidm_results_display`? This way the deletion of the folder is done just after the user has agreed on overwriting.